src/jdataview.js

```diff
@@ -289,29 +363,41 @@ export class jDataView {
 
 }
 
-const builtInTypes = [
-	"Float64", "Float32",
-	"BigInt64", "BigUint64",
-	"Int32", "Uint32",
-	"Int16", "Uint16",
-	"Int8", "Uint8"
-];
+const builtInTypeBytes = {
+	"Float64": 8, "Float32": 4,
+	"BigInt64": 8, "BigUint64": 8,
+	"Int32": 4, "Uint32": 4,
+	"Int16": 2, "Uint16": 2,
+	"Int8": 1, "Uint8": 1
+};
 // Encapsulate all the built-in methods
-for (const type of builtInTypes) {
+for (const type in builtInTypeBytes) {
+	const typeByteLength = builtInTypeBytes[type];
 	// Getters
 	jDataView.prototype["get" + type] = function (byteOffset, littleEndian) {
 		littleEndian = defined(littleEndian, this.littleEndian);
```

Comment:
Note: in most (but probably not all) places those `defined` helpers can be replaced with ES6 param defaults now.

Reply:
The rest can be replaced with `??` or `??=` operators, although it depends which browsers we want to cover... I guess those who need it can always transpile anyway.

Reply:
Yeah they'll get automagically transpiled by Vite. I entirely just forgot to swap them out!